### PR TITLE
Admin override fix

### PR DESCRIPTION
**Pull request by epritchard**
/cc @zendesk/spartan

### Description
This fixes an issue where the admin override setting was not being checked for when validating user input.

### Risks
* Low - validation could get wonky for edge cases, if I didn't test thoroughly enough

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -60,22 +60,24 @@
...
+      return this.currentUserRole === 'admin' && this.setting("adminOverride");
+    },
+
     getRemovableTags: function() {
-      var currentTags = this.ticket().tags(),
-        adminOverride = this.setting("adminOverride");
+      var currentTags = this.ticket().tags();
 
-      if (this.currentUserRole === 'admin' && adminOverride) {
+      if (this.adminOverride) {
```
Comment (shollinger) [inline app.js]: This needs to be `this.adminOverride()` as a function call, otherwise it will always return truthy because it's a defined function.

File: app.js [hunk 2]
```diff
@@ -60,22 +60,24 @@
...
       }
     },
 
     getRestrictedTags: function() {
-      var currentTags = this.ticket().tags(),
-        adminOverride = this.setting("adminOverride");
+      var currentTags = this.ticket().tags();
 
-      if (this.currentUserRole === 'admin' && adminOverride) {
+      if (this.adminOverride) {
```
Comment (shollinger) [inline app.js]: Ditto

Comment (epritchard) [inline app.js]: Good catches ... forgot to fix these! :facepalm:

**Comment by epritchard** [comment]
Good to go now @shollinger?

**Comment by shollinger** [comment]
👍
